cli/resource.go

```diff
@@ -72,3 +77,28 @@ func runResourceAdd(args *docopt.Args, client *controller.Client) error {
 
 	return nil
 }
+
+func runResourceRemove(args *docopt.Args, client *controller.Client) error {
+	provider := args.String["<provider>"]
+	resource := args.String["<resource>"]
+
+	res, err := client.DeleteResource(provider, resource)
+	if err != nil {
+		return err
+	}
+
+	// Unset all the keys associated with this resource
+	env := make(map[string]*string)
+	for k := range res.Env {
+		env[k] = nil
```

Comment:
Oh. Facepalm. Easy enough to do then. I will change it so it unsets only when the variable hasn't changed since provisioning.